addons/edi/models/edi_synchronizer.py

```diff
@@ -370,7 +381,10 @@ def execute(self):
                     vals_list = [rec.target_values(rec._record_values()) for rec in batch]
```

Comment:
Suggestion:
Maybe now we need to change _record_values to get only the _changed_vals